rescript/plugin_setup.py

```diff
@@ -550,13 +550,16 @@
              ('unmatched_seqs', FeatureData[Sequence])],
     input_descriptions={
         'sequences': 'Sequences to be oriented.',
-        'reference_sequences': 'Reference sequences to orient against.'},
+        'reference_sequences': ('Reference sequences to orient against. If no'
+                                ' reference is provided, all the sequences '
```

Comment:
can the space at the beginning of this line be moved to the end of the previous line? (makes for easier reading)